### PR TITLE
Update fastlane tests to work with Ruby 2.4.0

### PR DESCRIPTION
**Pull request by KrauseFx**
Via https://github.com/bblimke/webmock/issues/683

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Pull Request is Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane.gemspec [hunk 1]
```diff
@@ -73,7 +73,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry'
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'yard', '~> 0.8.7.4'
-  spec.add_development_dependency 'webmock', '~> 1.19.0'
+  spec.add_development_dependency 'webmock', '~> 2.3.2'
```
Comment (nafu) [inline fastlane.gemspec]: This Includes same change from https://github.com/fastlane/fastlane/pull/8538 ?

Comment (nafu) [inline fastlane.gemspec]: I think this branch should be rebased!

Comment (KrauseFx) [inline fastlane.gemspec]: yeah, no need to rebase since it's the same change anyway

Comment (nafu) [inline fastlane.gemspec]: I see 👍  I just feel that it would be nice for reviewing because the change become more clear, but it's ok!

**Review by ohayon — APPROVED** [review]
👍

**Pull request merged**